src/_pages-content/faq.md

```diff
@@ -0,0 +1,76 @@
+
+#### I want to present at the conference. What should I do?
+The call for Proposals is now open. Please visit the [CFP page](https://in.pycon.org/cfp/2024/proposals/) to submit your talk/workshop/ proposal.
+
+#### I am a first-time speaker & need some help in getting started. What should I do?
+Check out the [Guide to Speaking](https://in.pycon.org/blog/2023/guide-to-speaking.html) to understand more about the process. For any doubts, feel free to reach out to us!
+
+#### I want to volunteer for the conference. What should I do?
+Check out the [Volunteer Page](https://in.pycon.org/blog/2023/volunteering-guide-pyconin23.html) to know about the roles and responsibilities. Fill out the [volunteer form](https://docs.google.com/forms/d/e/1FAIpQLSdmlVnxDBENiXs17hQeUtgqOZdcsWI7lmQpXq5m9K-e0eS52Q/viewform) and subscribe to the [InPyCon mailing list](https://mail.python.org/mailman/listinfo/inpycon) for updates.
+
+#### Will the Internet and WiFi setup be good enough for a large crowd? Should I get a data card?
+We try really hard to arrange good Internet at the venue. Most of the time it is adequate. But the unpredictable amount of usage during the event sometimes causes problems. So it is fine if you don’t carry a data card, but there is no harm in being prepared for the worst.
+
+### Ticketing FAQ
+
+#### I am a speaker/volunteer at the conference. Do I need to buy a ticket?
+Yes. At PyCon India, everybody buys tickets. Even the event organizers buy tickets. Your ticket helps keep the event affordable for all.
+
+#### What does the Conference (September 21-22, 2024) ticket include?
+- A full conference pass for two days that includes access to all keynotes, talks, poster sessions, lightning talks, and open spaces.
+- Lunch
+- High-Tea and Snacks
+- PyCon India T-shirt and goodies
+
+#### Is there a different ticket for the Workshops?
+Yes. A separate ticket has to be purchased for each workshop. The following is included as part of the workshop ticket:
+- Access to the specific workshop
+- Lunch
+- High-Tea and Snacks
+- Access to Kid's engagement (by Volunteers) at the Venue
+
+#### Is there a different ticket for the Devsprints?
+No, you do not need a ticket for Devsprints. Devsprints are open to all participants and is free to attend.
+
+#### Is accommodation included in the ticket?
+No, it doesn’t include accommodation.
+
+#### Would there be any spot registration at the venue?
+No, there won’t be any spot registrations available at the conference venue.
```

Comment:
Yes there might be spot registrations as discussed on last call. Please refer to meeting notes.